pkg/cmd/cli/cmd/process.go

```diff
@@ -149,8 +168,8 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 		obj, ok := infos[i].Object.(*api.Template)
 		if !ok {
 			sourceName := filename
-			if len(storedTemplate) > 0 {
-				sourceName = namespace + "/" + storedTemplate
+			if len(templateName) > 0 {
+				sourceName = namespace + "/" + templateName
```

Comment:
that one is tricky :-) This basically allow to process multiple templates (iow. cat foo bar | oc process). We don't allow the '//' syntax for file input, so I think the "namespace" is OK here.